Makefile

```diff
@@ -16,7 +16,7 @@ setup:
 
 .PHONY: test
 test:
-	python -m coverage run -m advice_animal.tests $(TESTOPTS)
+	python -m coverage run -m pytest $(TESTOPTS)
```

Comment:
is this the recommended way, or to use `pytest-cov`?

Reply:
I think both ways are fine but `pytest-cov` requires an additional dependency. 🤷